code/src/components/Questions/CheckboxGroup.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const CheckboxGroup = ({ energy, onEnergyChange }) => {
+
+
+  return (
+    <div className="question-card">
+      <h3 className="question-header" tabIndex="0">How often du you push yourself outside of your comfort zone?</h3>
+      <label htmlFor="never">Never</label>
```

Comment:
It is  possible to check mutiple checkboxes that are displayed in the summary. Maybe add a blank space in between if many options are selected ? 
![image](https://user-images.githubusercontent.com/47081254/97582052-99073380-19f5-11eb-8ffd-982f5330c49a.png)